cnv/cnv_users_guide/cnv-ansible-modules.adoc

```diff
@@ -0,0 +1,15 @@
+[id="ansible-modules"]
+= Automating management tasks using Ansible
+include::modules/cnv-document-attributes.adoc[]
+:context: cnv-ansible-modules
+toc::[]
```

Comment:
```suggestion
```

Reply:
There isn't a table of contents in the module itself, so there's no need for `toc::[]`.